projects/plugins/jetpack/modules/masterbar/inline-help/inline-help.js

```diff
@@ -0,0 +1,22 @@
+( function () {
+	var windowObjectReference = null;
+	function init() {
+		document.addEventListener( 'click', function ( e ) {
```

Comment:
Since we're loading the `inline-help.js` script in the footer, why don't we optimize this selector and only listen for clicks on `data-faxu-inline-help` elements?

```
document.querySelector( '[data-faux-inline-help]' ).addEventListener( ... );
```